src/footer/css/footer.pcss

```diff
@@ -80,6 +87,89 @@
 /* Global Footer: Sections */
 /* ========================================================================== */
 
+.esri-footer-section--0 {
+
+	@media (--gfoot-l) {
+		padding-left: calc(50% - 1424px / 2);
+		padding-right: calc(50% - 1424px / 2);
+	}
+
+}
+
+.esri-footer-breadcrumb {
+	box-sizing: border-box;
+	padding: 25px;
+}
+
+.clearFloats {
```

Comment:
instead of having this `clearFloats` class, I would clear the floats as a pseudo element on the parent, like:


```
.esri-footer-section--0:after {
  content: "";
  display: table;
  clear: both;
}
```

Then you can remove this class and the empty div you're adding in the js file

Reply:
You got it..